api4/status_test.go

```diff
@@ -92,6 +92,18 @@ func TestGetUsersStatusesByIds(t *testing.T) {
 		assert.Equal(t, "dnd", userStatus.Status)
 	}
 
+	invalidUserIds := []string{"invalid_user_id", "invalid_user_id"}
+	_, resp = Client.GetUsersStatusesByIds(invalidUserIds)
+	CheckBadRequestStatus(t, resp)
+
+	invalidUserIds = []string{th.BasicUser.Id, "invalid_user_id"}
+	_, resp = Client.GetUsersStatusesByIds(invalidUserIds)
+	CheckBadRequestStatus(t, resp)
+
+	invalidUserIds = []string{"", ""}
```

Comment:
Would be great to have another test with an empty list of userIds.

Reply:
added it, thank you for the review!

Reply:
That was fast! Thanks!